### PR TITLE
Resolves #1382

### LINKED ISSUE #1382: Example 1 has confusing text and spacing doesn't help explain it
Example 1 currently shows the following, but with very strange spacing making it hard to follow:
-----------------
Set-NetIPsecRule -DisplayName "Tunnel Rule" -LocalAddress Any
This task can be alternatively done with the following cmdlets.
$nfwAddressFilter = ( Get-NetIPsecRule -DisplayName "Tunnel Rule" | Get-NetFirewallAddressFilter )
Set-NetFirewallAddressFilter -InputObject $nfwAddressFilter -LocalAddress Any
This task can be alternatively done with the following cmdlet.
Get-NetIPsecRule -DisplayName "Tunnel Rule" | Get-NetFirewallAddressFilter | Set-NetFirewallAddressFilter -LocalAddress Any
-----------------

What is important to realise is that there are THREE ways that this one task is being explained. The first is a blanket "Set-"; the second (and least clear part of the example) is to "Get-" the rule and its filter, then to "Set-" the filter based on the input object from the "Get-"; the third process described is a full pipeline from Get- to Set-.

It may be beneficial to better group the related commands in the second approach to this task, so it's clear they are a combined unit. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 08987588-5154-1f17-ca21-981a6df9d24e
* Version Independent ID: 53c5f926-9e96-226b-5456-db39e6fcf468
* Content: [Set-NetFirewallAddressFilter (netsecurity)](https://docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewalladdressfilter?view=win10-ps)
* Content Source: [docset/windows/netsecurity/set-netfirewalladdressfilter.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/windows/netsecurity/set-netfirewalladdressfilter.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix all example formatting and adjust links and descriptions in all version of the article.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1382

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 1]
```diff
@@ -209,52 +194,36 @@ Accept wildcard characters: False
...
-
- ---- Active Directory GPOs can be created using the New-GPO cmdlet or the Group Policy Management Console.
-  
-
- -- RSOP: This read-only store contains the sum of all GPOs applied to the local computer. 
-
- -- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server® 2012. 
-
- -- StaticServiceStore: This read-only store contains all the service restrictions that ship with Windows Server 2012.
+------ `-PolicyStore hostname`.
```
Comment (tseward) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]: It looks like this section didn't format correctly. All the text runs on a single line.

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 2]
```diff
@@ -42,48 +42,36 @@ Set-NetFirewallAddressFilter -InputObject <CimInstance[]> [-LocalAddress <String
...
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
+```powershell
 PS C:\>Set-NetIPsecRule -DisplayName "Tunnel Rule" -LocalAddress Any
 
-
-This task can be alternatively done with the following cmdlets.
+# This task can be alternatively done with the following cmdlets.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively this task can be done with the following cmdlets.
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 3]
```diff
@@ -180,15 +175,16 @@ Specifies that network packets with matching IP addresses match this rule.
...
 - Single IPv6 Address: fe80::1 
 - IPv4 Subnet (by network bit count): 1.2.3.4/24 
 - IPv6 Subnet (by network bit count): fe80::1/48 
 - IPv4 Subnet (by network mask):  1.2.3.4/255.255.255.0 
 - IPv4 Range: 1.2.3.4 through 1.2.3.7 
 - IPv6 Range: fe80::1 through fe80::9 
-Querying for rules with this parameter can only be performed using filter objects.
-See the Get-NetFirewallAddressFilter cmdlet for more information.
+
+Note: Querying for rules with this parameter can only be performed using filter objects.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
> [!NOTE]
> Querying for rules with this parameter can only be performed using filter objects.
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 4]
```diff
@@ -180,15 +175,16 @@ Specifies that network packets with matching IP addresses match this rule.
...
+See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 5]
```diff
@@ -271,8 +267,9 @@ The acceptable formats for this parameter are:
 - IPv4 Subnet (by network mask): 1.2.3.4/255.255.255.0 
 - IPv4 Range: 1.2.3.4 through 1.2.3.7 
 - IPv6 Range: fe80::1 through fe80::9 
-Querying for rules with this parameter can only be performed using filter objects.
-See the Get-NetFirewallAddressFilter cmdlet for more information.
+
+Note: Querying for rules with this parameter can only be performed using filter objects.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
> [!NOTE]
> Querying for rules with this parameter can only be performed using filter objects. See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 6]
```diff
@@ -271,8 +267,9 @@ The acceptable formats for this parameter are:
...
+See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md (hunk 2 above)
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlets.
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 7]
```diff
@@ -42,48 +42,36 @@ Set-NetFirewallAddressFilter -InputObject <CimInstance[]> [-LocalAddress <String
...
+# This task can be alternatively done with the following cmdlets.
 PS C:\>$nfwAddressFilter = ( Get-NetIPsecRule -DisplayName "Tunnel Rule" | Get-NetFirewallAddressFilter )
-
-
-
 PS C:\>Set-NetFirewallAddressFilter -InputObject $nfwAddressFilter -LocalAddress Any
 
-
-This task can be alternatively done with the following cmdlet.
+# This task can be alternatively done with the following cmdlet.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlet.
```

File: docset/windows/netsecurity/Set-NetFirewallAddressFilter.md [hunk 8]
```diff
@@ -42,48 +42,36 @@ Set-NetFirewallAddressFilter -InputObject <CimInstance[]> [-LocalAddress <String
...
-
 PS C:\>$nfwAddressFilterLS6 = ( Where-Object -InputObject $nfwAddressFilter -Property { $_.RemoteAddress -Eq "LocalSubnet6" } )
-
-
-
 PS C:\>Set-NetFirewallAddressFilter -InputObject $nfwAddressFilterLS6 -RemoteAddress LocalSubnet4
 
-
-This task can be alternatively done with the following cmdlet.
+# This task can be alternatively done with the following cmdlet.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlet.
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 9]
```diff
@@ -32,50 +32,38 @@ Set-NetFirewallAddressFilter [-AsJob] [-CimSession <CimSession[]>] [-LocalAddres
...
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
+```powershell
 PS C:\>Set-NetIPsecRule -DisplayName "Tunnel Rule" -LocalAddress Any
 
-
-This task can be alternatively done with the following cmdlets.
+# This task can be alternatively done with the following cmdlets.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlets.
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 10]
```diff
@@ -32,50 +32,38 @@ Set-NetFirewallAddressFilter [-AsJob] [-CimSession <CimSession[]>] [-LocalAddres
...
 PS C:\>$nfwAddressFilter = ( Get-NetIPsecRule -DisplayName "Tunnel Rule" | Get-NetFirewallAddressFilter )
-
-
-
 PS C:\>Set-NetFirewallAddressFilter -InputObject $nfwAddressFilter -LocalAddress Any
 
-
-This task can be alternatively done with the following cmdlet.
+# This task can be alternatively done with the following cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlet.
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 11]
```diff
@@ -32,50 +32,38 @@ Set-NetFirewallAddressFilter [-AsJob] [-CimSession <CimSession[]>] [-LocalAddres
...
-
 PS C:\>$nfwAddressFilterLS6 = ( Where-Object -InputObject $nfwAddressFilter -Property { $_.RemoteAddress -Eq "LocalSubnet6" } )
-
-
-
 PS C:\>Set-NetFirewallAddressFilter -InputObject $nfwAddressFilterLS6 -RemoteAddress LocalSubnet4
 
-
-This task can be alternatively done with the following cmdlet.
+# This task can be alternatively done with the following cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlet.
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 12]
```diff
@@ -209,52 +194,36 @@ Accept wildcard characters: False
...
- -- StaticServiceStore: This read-only store contains all the service restrictions that ship with Windows Server 2012.
+------ `-PolicyStore hostname`. 
+---- Active Directory GPOs can be specified as follows. 
+------ `-PolicyStore domain.fqdn.com\GPO_Friendly_Namedomain.fqdn.comGPO_Friendly_Name`. 
+------ Such as the following. 
+-------- `-PolicyStore localhost`
+-------- `-PolicyStore corp.contoso.com\FirewallPolicy`
+---- Active Directory GPOs can be created using the **New-GPO** cmdlet or the Group Policy Management Console. 
+- RSOP: This read-only store contains the sum of all GPOs applied to the local computer. 
+- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server® 2012.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server 2012. 
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 13]
```diff
@@ -41,50 +41,38 @@ Set-NetFirewallAddressFilter -InputObject <CimInstance[]> [-LocalAddress <String
...
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
+```powershell
 PS C:\>Set-NetIPsecRule -DisplayName "Tunnel Rule" -LocalAddress Any
 
-
-This task can be alternatively done with the following cmdlets.
+# This task can be alternatively done with the following cmdlets.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlets.
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 14]
```diff
@@ -41,50 +41,38 @@ Set-NetFirewallAddressFilter -InputObject <CimInstance[]> [-LocalAddress <String
...
 PS C:\>$nfwAddressFilter = ( Get-NetIPsecRule -DisplayName "Tunnel Rule" | Get-NetFirewallAddressFilter )
-
-
-
 PS C:\>Set-NetFirewallAddressFilter -InputObject $nfwAddressFilter -LocalAddress Any
 
-
-This task can be alternatively done with the following cmdlet.
+# This task can be alternatively done with the following cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlet.
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 15]
```diff
@@ -41,50 +41,38 @@ Set-NetFirewallAddressFilter -InputObject <CimInstance[]> [-LocalAddress <String
...
-
 PS C:\>$nfwAddressFilterLS6 = ( Where-Object -InputObject $nfwAddressFilter -Property { $_.RemoteAddress -Eq "LocalSubnet6" } )
-
-
-
 PS C:\>Set-NetFirewallAddressFilter -InputObject $nfwAddressFilterLS6 -RemoteAddress LocalSubnet4
 
-
-This task can be alternatively done with the following cmdlet.
+# This task can be alternatively done with the following cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
# Alternatively, this task can be done with the following cmdlet.
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 16]
```diff
@@ -233,48 +218,32 @@ Accept wildcard characters: False
...
- -- StaticServiceStore: This read-only store contains all the service restrictions that ship with Windows Server 2012.
+------ `-PolicyStore hostname`. 
+---- Active Directory GPOs can be specified as follows. 
+------ `-PolicyStore domain.fqdn.com\GPO_Friendly_Namedomain.fqdn.comGPO_Friendly_Name`. 
+------ Such as the following. 
+-------- `-PolicyStore localhost`
+-------- `-PolicyStore corp.contoso.com\FirewallPolicy`
+---- Active Directory GPOs can be created using the **New-GPO** cmdlet or the Group Policy Management Console. 
+- RSOP: This read-only store contains the sum of all GPOs applied to the local computer. 
+- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server® 2012.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server 2012. 
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 17]
```diff
@@ -156,28 +150,19 @@ Accept wildcard characters: False
...
- -- IPv4 Subnet (by network bit count): 1.2.3.4/24 
-
- -- IPv6 Subnet (by network bit count): fe80::1/48 
-
- -- IPv4 Subnet (by network mask):  1.2.3.4/255.255.255.0 
-
- -- IPv4 Range: 1.2.3.4 through 1.2.3.7 
-
- -- IPv6 Range: fe80::1 through fe80::9 
 Note: Querying for rules with this parameter can only be performed using filter objects.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
> [NOTE]
> Querying for rules with this parameter can only be performed using filter objects. See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md (hunk 1 above)
Comment (velkovb) [reply docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]: This one is seriously messed up... I will try to fix it,

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 18]
```diff
@@ -156,28 +150,19 @@ Accept wildcard characters: False
...
-See the Get-NetFirewallAddressFilter cmdlet for more information.
+See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 19]
```diff
@@ -266,28 +235,19 @@ Accept wildcard characters: False
...
- -- IPv4 Subnet (by network bit count): 1.2.3.4/24 
-
- -- IPv6 Subnet (by network bit count): fe80::1/48 
-
- -- IPv4 Subnet (by network mask): 1.2.3.4/255.255.255.0 
-
- -- IPv4 Range: 1.2.3.4 through 1.2.3.7 
-
- -- IPv6 Range: fe80::1 through fe80::9 
 Note: Querying for rules with this parameter can only be performed using filter objects.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
> !NOTE]
> Querying for rules with this parameter can only be performed using filter objects. See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```

File: docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 20]
```diff
@@ -266,28 +235,19 @@ Accept wildcard characters: False
...
-See the Get-NetFirewallAddressFilter cmdlet for more information.
+See the **Get-NetFirewallAddressFilter** cmdlet for more information
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 21]
```diff
@@ -180,28 +174,19 @@ Accept wildcard characters: False
...
- -- IPv6 Range: fe80::1 through fe80::9 
+-- Single IPv4 Address: 1.2.3.4 
+- Single IPv6 Address: fe80::1 
+- IPv4 Subnet (by network bit count): 1.2.3.4/24 
+- IPv6 Subnet (by network bit count): fe80::1/48 
+- IPv4 Subnet (by network mask):  1.2.3.4/255.255.255.0 
+- IPv4 Range: 1.2.3.4 through 1.2.3.7 
+- IPv6 Range: fe80::1 through fe80::9 
+
 Note: Querying for rules with this parameter can only be performed using filter objects.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
> [!NOTE]
> Querying for rules with this parameter can only be performed using filter objects. See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 22]
```diff
@@ -180,28 +174,19 @@ Accept wildcard characters: False
...
-See the Get-NetFirewallAddressFilter cmdlet for more information.
+See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
```

Comment (velkovb) [reply docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]: Why you want to remove the line?

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 23]
```diff
@@ -290,28 +259,19 @@ Accept wildcard characters: False
...
- -- IPv6 Range: fe80::1 through fe80::9 
+- Single IPv4 Address: 1.2.3.4 
+- Single IPv6 Address: fe80::1 
+- IPv4 Subnet (by network bit count): 1.2.3.4/24 
+- IPv6 Subnet (by network bit count): fe80::1/48 
+- IPv4 Subnet (by network mask): 1.2.3.4/255.255.255.0 
+- IPv4 Range: 1.2.3.4 through 1.2.3.7 
+- IPv6 Range: fe80::1 through fe80::9 
+
 Note: Querying for rules with this parameter can only be performed using filter objects.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
> [!NOTE]
 Querying for rules with this parameter can only be performed using filter objects. See the **Get-NetFirewallAddressFilter** cmdlet for more information.
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md [hunk 24]
```diff
@@ -290,28 +259,19 @@ Accept wildcard characters: False
...
-See the Get-NetFirewallAddressFilter cmdlet for more information.
+See the **Get-NetFirewallAddressFilter** cmdlet for more information
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]:
```suggestion
```

File: docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md (hunk 22 above)
Comment (JohanFreelancer9) [reply docset/winserver2012r2-ps/netsecurity/Set-NetFirewallAddressFilter.md]: I have now included this line with my suggestion to format the note correctly.
